src/SortMovies/SortMovies.js

```diff
@@ -81,42 +75,38 @@ export class SortMovies extends Component {
                 <>{!this.state.favorites &&
                     <>
                         <NavLink className="sort-btn" to={`/favorites`}
-                            // {/* <button  */}
                                 type="apple" 
                                 role="apple"
                                 className="sort-btn"
                                 name="sort"
                                 onClick={this.toggleFavorites}
                                 >
                                 Favorites
-                            {/* </button> */}
                         </NavLink>
                     </>
                     }
                     {this.state.favorites && 
                     <>
                     <NavLink className="sort-btn" to={`/`}
-                        // {/* <button  */}
                             type="apple" 
                             role="apple"
                             className="sort-btn"
                             name="sort"
                             onClick={this.toggleFavorites}
                             >
                             All
-                            
-                        {/* </button> */}
                     </NavLink>
                     </>
                     }
                 </>
                 }
-              </div>          
+              </div>
             </div>
         )
     }
 }
 
 SortMovies.propType = {
-    sortMovies: PropTypes.func
+    sortMovies: PropTypes.func,
+    status: PropTypes.string
 }
```

Comment:
Let's also make a point tomorrow where we don't edit any more of the non-test files, then go back and double check all the PropTypes.